media-gfx/nifskope/nifskope-2.0.dev9.recipe

```diff
@@ -0,0 +1,111 @@
+Civilization IV, and more."
+HOMEPAGE="http://www.niftools.org/"
+COPYRIGHT="2016-2023 NifTools project"
+LICENSE="LICENSE"
+REVISION="1"
+SOURCE_URI="https://github.com/hexabits/nifskope/archive/refs/tags/v2.0.dev9.tar.gz"
+CHECKSUM_SHA256="b40baca5fc3b11292cd284b5ef0d07dbe40c3bab8067c05b7f25bdb6141fe26c"
+SOURCE_URI_2="https://github.com/g-truc/gli/archive/refs/tags/0.8.2.0.tar.gz"
+CHECKSUM_SHA256_2="9e7024c2df77c011eff4f66667c1834620c70b7902cd50f32ab48edd49fe0139"
+SOURCE_URI_3="https://github.com/qhull/qhull/archive/refs/tags/v8.1-alpha1.tar.gz"
+CHECKSUM_SHA256_3="09e5e4c5b2b8a9e617a46876fef5a3d33e70aa1d08a163ff05d37701327c3be7"
+SOURCE_URI_4="https://github.com/niftools/nifxml/archive/refs/tags/v0.9.0.tar.gz"
+CHECKSUM_SHA256_4="984c247115bc49f90ded69b4c8feb9a62a0365ad53f5890312302b6a05bb4ed7"
+ADDITIONAL_FILES="nifskope.rdef.in"
+
+PATCHES="nifskope-2.0.dev9.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+    settings/nifskope/kfm.xml keep-old
+    settings/nifskope/nif.xml keep-old	
+    "
+	
+PROVIDES="
+    nifskope$secondaryArchSuffix = $portVersion
+    app:NifSkope$secondaryArchSuffix = $portVersion
+    app:nif.xml$secondaryArchSuffix = $portVersion
+    app:kfm.xml$secondaryArchSuffix = $portVersion
+    "
+	
+REQUIRES="
+    haiku$secondaryArchSuffix
+    lib:libGLU$secondaryArchSuffix
+    lib:libQt5Core$secondaryArchSuffix
+    lib:libGL$secondaryArchSuffix
+    lib:libz$secondaryArchSuffix    
+    lib:libQt5Network$secondaryArchSuffix
+    lib:libQt5OpenGL$secondaryArchSuffix
+    lib:libQt5Gui$secondaryArchSuffix
+    lib:libQt5Xml$secondaryArchSuffix
+    lib:libQt5Widgets$secondaryArchSuffix
+    "
+
+BUILD_REQUIRES="
+    haiku${secondaryArchSuffix}_devel
+    devel:glm$secondaryArchSuffix	
+    devel:libqhull_r$secondaryArchSuffix
+    devel:libz$secondaryArchSuffix
+    devel:libGLU$secondaryArchSuffix
+    "
+
+BUILD_PREREQUIRES="
+    cmd:g++$secondaryArchSuffix
+    cmd:clang++$secondaryArchSuffix
+    cmd:make
+    cmd:pkg_config$secondaryArchSuffix
+    cmd:qmake$secondaryArchSuffix >= 5
+    cmd:which
+    cmd:wget
+    "
+
+BUILD()
+{
+    cp -r $sourceDir2/gli-0.8.2.0/gli/* $sourceDir/src/
+    cp -r $sourceDir2/gli-0.8.2.0/external/glm/* $sourceDir/src/
+    cp -r $sourceDir3/qhull-8.1-alpha1/src/libqhull $sourceDir/src/
+    mkdir -p $sourceDir/build/docsys/nifxml/
+    cp $sourceDir4/nifxml-0.9.0/nif.xml $sourceDir/build/docsys/nifxml/
+    mkdir -p $sourceDir/build/docsys/kfmxml/
+    wget https://raw.githubusercontent.com/niftools/kfmxml/develop/kfm.xml
+    cp ./kfm.xml $sourceDir/build/docsys/kfmxml/
+    qmake
+    make $jobArgs
+}
+
+INSTALL()
+{
+    make install
+
+    mkdir -p $appsDir
+    mkdir -p $appsDir/../settings/
+    mkdir -p $appsDir/../settings/nifskope/
+
+    cp $sourceDir/release/NifSkope $appsDir/
+    cp $sourceDir/release/nif.xml $appsDir/../settings/nifskope/
+    cp $sourceDir/release/kfm.xml $appsDir/../settings/nifskope/
```

Comment:
For installing into /system/settings you can use $settingsDir here instead of $appsDir/../settings, line 86 and 87 can be combined as: "mkdir -p $settingsDir/nifskope" (mind the typos if any) :)

Reply:
$settingsDir is now used

Reply:
Ok fixed